src/codec.ts

```diff
@@ -171,18 +179,18 @@ export function bytesToString (buf: Uint8Array): string {
 /**
  * String -> Uint8Array
  */
-export function stringToBytes (str: string): Uint8Array {
+export function stringToBytes (str: string): { bytes: Uint8Array, tuples: Tuple[], path: string | null } {
```

Comment:
As this function not jsut converting bytes, can we name it more appropriate e.g. `parseTuples`

Reply:
please have a look at the new type and function name

```typescript
export function parseMultiaddrInputString (str: MultiaddrInputString): MultiaddrInputStringResult
```